examples/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json

```diff
@@ -24,7 +24,7 @@
   "components": [
     {
       "id": "prostate-learner",
-      "path": "pt.learners.supervised_monai_prostate_learner.SupervisedMonaiProstateLearner",
```

Comment:
I think `pt.` is required if the script is started from the `brats18` folder and the PYTHONPATH is set as [here](https://github.com/NVIDIA/NVFlare/blob/9a1b226bfb507fce8ea5a1cf3fa15c2a00946806/examples/brats18/start_fl_poc.sh#L3).

Reply:
in past version these were put under pt/, but recent version we changed to custom/ , so need to removed pt. to make the path recognizable.